examples/ca-tel.html

```diff
@@ -0,0 +1,83 @@
+<!doctype>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>CA TEL | Web Lab Common UI</title>
+        <script src="/bower_components/system.js/dist/system.js"></script>
+        <script src="/components/config/index.js"></script>
+        <link rel="stylesheet" type="text/css" href="/examples/index.css">
+        <link rel="stylesheet" type="text/css" href="/components/ca-select.css">
+        <link rel="stylesheet" type="text/css" href="/components/ca-tel.css">
+    </head>
+
+    <body>
+        <div class="ca-home">
+            <a id="home" href="../"></a>
+        </div>
+
+        <div class="ca-title">CA TEL EXAMPLE</div>
+        <ca-tel></ca-tel>
+
+        <script>
+            SystemJS.import('ca-tel.js').then(() => {
+                console.log(document.querySelector('ca-tel'));
+                SystemJS.import('helpers/component-support.js').then((componentSupport) => {
```

Comment:
don't need the brackets around single input variable, so should be:
.then(componentSupport => {